drupalmel.info

```diff
@@ -31,6 +31,12 @@ nodes[realityloop][type] = blog_feed
 nodes[realityloop][feeds][FeedsHTTPFetcher][source] = http://realityloop.com/tags/drupal-planet/feed
 nodes[realityloop][language] = und
 
+nodes[realityloop][uid] = 1
```

Comment:
The 'realityloop' part of this is a unique key of an array, so this entry will actually overwrite the previous entry. Will need to change it to 'flink'